wavefront-istio-adapter/values.yaml

```diff
@@ -0,0 +1,25 @@
+adapter:
+  image: vmware/wavefront-adapter-for-istio
+  tag: 0.1.3
+
+credentials:
+# Define either direct or proxy credentials.
+  direct:
+    server: https://YOUR-INSTANCE.wavefront.com
+    token: YOUR-API-TOKEN
+#  proxy:
+#    address: YOUR-PROXY-IP:YOUR-PROXY-PORT
+
+namespaces:
+  istio: istio-system
+  adapter: wavefront-istio
```

Comment:
The adapter namespace can be specified via the helm command. We should let helm handle this. This will require removing the namespace references in the yaml files.